models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+class Department(Base):
+    __tablename__ = 'departments'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(100), nullable=False)
+    employees = relationship("Employee", back_populates="department")
+
+class Employee(Base):
+    __tablename__ = 'employees'
+
+    id = Column(Integer, primary_key=True)
+    first_name = Column(String(100), nullable=False)
+    last_name = Column(String(100), nullable=False)
+    salary = Column(Float, nullable=False)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding salary field validation</b></div>
<div id="fix">

Consider adding validation for the `salary` field to ensure it's not negative. You might want to use a `CheckConstraint` or implement a validator function.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -20 +20,21 @@
 -    salary = Column(Float, nullable=False)
 +    salary = Column(Float, nullable=False, CheckConstraint('salary >= 0'))

```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/12#issuecomment-2514850927>#ad7e48</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged